### PR TITLE
Add Luna Studio latest

### PR DESCRIPTION
**Pull request by hanxue**
I need help to create this Cask. The file http://packages.luna-lang.org/darwin/lunaInstaller.zip contains `lunaInstaller.app`, which will then select the version of Luna Studio to download and install. 

Is the current content of the Luna Studio cask acceptable?

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

### CODE REVIEW

**Comment by csi-adziahel** [comment]
Guess that it should run the installer then actually setting up the Luna Studio, shouldn't it?

There's `consoleInstaller` binary under `lunaInstaller.app/Contents/Resources/app` which you should be able to run (`./consoleInstaller --batch install --version=#{version}` should probably do the job). Or you could get link to the _studio_ tarball from http://packages.luna-lang.org/config.yaml and install it directly.

**Comment by commitay** [comment]
Keeping this as `installer manual` rather than running the binary which downloads the app or fetch the tarball which looks like it need additional deps installed.

**Review by commitay — APPROVED** [review]

File: Casks/lunastudio.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  version :latest
+  sha256 :no_check
+
+  url 'http://packages.luna-lang.org/darwin/lunaInstaller.zip'
+  name 'Luna Studio'
+  homepage 'http://www.luna-lang.org/'
+
+  installer manual: 'lunaInstaller.app'
+
+  uninstall delete: '/Applications/LunaStudio.app'
```
Comment (vitorgalvao) [inline Casks/lunastudio.rb]: It’s unlikely this is enough to uninstall the package.

**Review by vitorgalvao — DISMISSED** [review]

File: Casks/lunastudio.rb [hunk 2]
```diff
@@ -8,5 +8,10 @@
 
   installer manual: 'lunaInstaller.app'
 
-  uninstall delete: '/Applications/LunaStudio.app'
+  uninstall delete: [
+                      '/Applications/LunaStudio.app',
+                      '~/.luna/config/luna-studio',
```
Comment (commitay) [inline Casks/lunastudio.rb]: The installed packages are in this folder.
```
236M	/Users/commitay/.luna/config/luna-studio
```

The `~/.luna` folder only has a couple of config files.

**Pull request merged**